### PR TITLE
Umlaut Fixes

### PR DESCRIPTION
**Pull request by kevinreiss**
This is a first pass.

### CODE REVIEW

File: app/views/catalog/_show_main_content.html.erb [hunk 1]
```diff
@@ -19,14 +19,14 @@
   <script type="text/javascript">
     jQuery(document).ready(function(){
       var ctx = $("span.Z3988").attr("title");
-      var umlaut_base = "https://getit-dev.princeton.edu";
+      var umlaut_base = "https://getit.princeton.edu";
```
Comment (kevinreiss) [inline app/views/catalog/_show_main_content.html.erb]: This should be configured somewhere if we are going to continue to use it.

File: config/initializers/blacklight_initializer.rb [hunk 2]
```diff
@@ -24,39 +24,57 @@ def export_as_openurl_ctx_kev(format = nil)
     id = to_marc.find{|field| field.tag == '001'}
```
Comment (kevinreiss) [inline config/initializers/blacklight_initializer.rb]: The idea is that we will move the entire export_as_openurl_ctx_kev function logic (and any attendant helper code) to orangeindex and just inject an OpenURL into the solr document. Correct @tampakis.

Comment (tampakis) [inline config/initializers/blacklight_initializer.rb]: Correct.

Comment (kevinreiss) [inline config/initializers/blacklight_initializer.rb]: Should I make an issue in orangeindex?

Comment (tampakis) [inline config/initializers/blacklight_initializer.rb]: I don't know if this is ideal, but we've been opening the orangeindex issues in orangelight.

Comment (kevinreiss) [inline config/initializers/blacklight_initializer.rb]: Okay. I'll open one.

File: app/views/catalog/_show_main_content.html.erb (hunk 1 above)
Comment (tampakis) [inline app/views/catalog/_show_main_content.html.erb]: Maybe in the [blacklight.yml](https://github.com/pulibrary/orangelight/blob/development/config/blacklight.yml)?

**Pull request merged**